### PR TITLE
Event Logic  feature

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
## Changes made include:

- [ ]  calculate.js and operate.js into a logic/ directory 
- [ ] Added big.js from https://www.npmjs.com/package/big-js as a dependency.
- [ ] Imported calculate.js and operate.js files into the Calculator component.
- [ ] Implemented the event handlers to use the math logic from calculate.js and operate.js in the project React component.
- [ ]  used state and props, passed props to child components, used lifecycle methods, lift state.
- [ ] Tested app in the browser with results below:
readme

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,176 +1,128 @@
...
+        <Button
+          typeBtn="0"
+          btnOnClick={this.btnOnClick}
+          btnClass="btn-light btn-zero"
+        />
+        <Button
+          typeBtn="."
+          btnOnclick={this.btnOnClick}
+          btnClass="btn-light btn-dot"
+        />
```
Comment (VICTOR-CHUKWU) [inline src/components/Calculator.js]: - [ ] currently this button is not clicking, kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Shedrack-Sunday  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (peterrobert) [reply src/components/Calculator.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Shedrack-Sunday , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**